app/src/test/java/com/example/bottomnav/ParseTest.java

```diff
@@ -126,4 +124,56 @@ public void structNConst() throws Exception {
         assertEquals("uint16_t", test.getConst("CAN_ID3").type);
         assertEquals("0x64", test.getConst("CAN_ID3").value);
     }
+
+    @Test
+    public void parseData() throws Exception {
+        Optional<Parse> open = Parse.parseTextFile("parseData.txt");
```

Comment:
Recommend naming the file with a `.h` extension for a C++ header file.
It shouldn't affect your code, but is a tad more realistic than `.txt`.